pages/dashboard.tsx

```diff
@@ -0,0 +1,335 @@
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+    {
+      createdAt: new Date().toISOString(),
+      transactionId: "tnbesc900023840-2i",
+      role: Role.buyer,
+      user: "Mr Sky",
+      agent: "Player-X",
+      rate: 0.02,
+      amount: 250000,
+      paymentIn: "btc",
+    },
+  ];
+
+  return (
+    <>
+      <Layout>
+        <div className="font-mono py-12 px-12 grid bg-gray-100 md:grid-cols-4 md:grid-rows-5 grid-cols-1 gap-4">
+          <div className="text-sm p-6 bg-white text-primary shadow-md md:row-start-1 md:row-span-5 flex flex-col space-y-4">
+            <div className="text-center flex flex-col  items-center w-full">
+              <img
+                src="http://tinygraphs.com/labs/isogrids/hexa16/TNB Crow?theme=seascape&numcolors=4&size=220&fmt=svg"
+                alt="Avatar"
+                className="rounded-full border-primary border-2 p-1 w-20"
+              />
+              <span>
+                <b>Bill Lakes</b>
+              </span>
+            </div>
+            <span className="text-center">
+              Trustworthy, fast, reliable. no scam.
+            </span>
+            <Button>Wallet</Button>
+
+            <div className="h-full flex flex-col justify-between">
+              <div className="md:space-y-8 md:mt-8 flex md:flex-col justify-around flex-row mb-4">
+                <div className="text-left space-y-1 flex flex-col m sm:w-auto w-28">
+                  <h4>
+                    <b>Verification</b>
+                  </h4>
+                  <span>
+                    Phone number <b>verified</b>
```

Comment:
for bold, you can use className="font-bold"

Reply:
Thanks